src/core/GizmoHelper.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react'
+import { createPortal, useFrame, useThree } from 'react-three-fiber'
+import { Group, Intersection, Matrix4, Object3D, Quaternion, Raycaster, Scene, Vector3 } from 'three'
+import { OrthographicCamera } from './OrthographicCamera'
+import { useCamera } from './useCamera'
+
+type GizmoHelperContext = {
+  tweenCamera: (direction: Vector3) => void
+  raycast: (raycaster: Raycaster, intersects: Intersection[]) => void
+}
+
+const Context = React.createContext<GizmoHelperContext>({} as GizmoHelperContext)
+
+export const useGizmoHelper = () => {
```

Comment:
This should probably be called `useGizmoContext` no? because you're just returning the context....